docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -206,6 +206,27 @@ more information.)
 
 ![DynamoDB local connection in NoSQL Workbench](../../img/database-access/nosql-workbench-connection.png)
 
+## SQL Server with Azure Data Studio
+
+In Azure Data Studio create a connection using your proxy's endpoint. This is
+`localhost,62652` in the example above. On a Windows machine, using an address in the format `127.0.0.1,62652` could be required instead of `localhost`. (See
+[Get connection information](./gui-clients.mdx#get-connection-information) for
+more information.)
+
+Create a connection with Microsoft SQL Server with these settings:
+
+|Connection Detail|Value|
+|---|---|
+|Server|`host`,`port` of proxy endpoint|
+|Authentication Type|SQL Login|
+|Password|empty|
+|Encrypt|`False`|
+
+Example:
+![Azure Data Studio connection options](../../img/database-access/azure-data-studio-connection.png)
+
+Click Connect to connect.
```

Comment:
This sentence seems redundant and/or obvious. I suggest removing it. If not, stylize the reference to the button:

```suggestion
Click **Connect** to connect.
```